generative/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -22,15 +22,21 @@
    "id": "777b7dcb",
```

Comment:
<p>Let's leave this on as "Setup environment" to be consistent with the other notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks. @KumoLiu I made according changes. Please proceed.